src/libs/isReportActionAttachment.js

```diff
@@ -0,0 +1,14 @@
+import _ from "underscore";
+import lodashGet from "lodash/get";
+import isReportMessageAttachment from "./isReportMessageAttachment";
+
+/**
+ * Check whether a report action is an attachment.
+ *
+ * @param {Object} reportAction report action
+ * @returns {Boolean}
+ */
+export default function isReportActionAttachment(reportAction) {
+    const message = _.first(lodashGet(reportAction, 'message', [{}]));
```

Comment:
I switched to `first` to be consistent with the `['message', 0, 'html']` path I used elsewhere, but attachment messages (and comment messages in general) can't have multiple fragments if I understand this correctly.

Reply:
[See also](https://github.com/Expensify/App/pull/25560/files#r1299852118)

Reply:
What I meant is that...
...in some places, [like this](https://github.com/Expensify/App/blob/64ecc7a21760f4d1e7356058f924fe8eb7023e88/src/libs/ReportUtils.js#L1445), we were looking at the **last** fragment of the attachment message (this is what I de-duplicate)
...but in other places, [like this](https://github.com/Expensify/App/blob/64ecc7a21760f4d1e7356058f924fe8eb7023e88/src/libs/ReportUtils.js#L1446) (actually the next line), we were looking at the **first** fragment of the attachment message

In both cases, the actual assumption seems to be that an attachment message has only one fragment. Still, I decided to make that consistent and always look at the first fragment.

By "fragment", I mean an element of the `message` array.

Reply:
@venture1981 Let me know if my explanation clarified this

Reply:
@venture1981 As the merge freeze is over, it would be great if we could resolve this conversation. If for some reason you're not available to reply, I'll resolve this thread anyway and try to get this PR merged. Thanks!